package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,14 @@
+-------------------------------------------------------------------
+Tue Aug 28 13:27:53 UTC 2018 - ancor@suse.com
+
+- Fixed a wrong unit test.
+- 4.1.10
```

Comment:
NP: I would add the version number comment in the previous entry (to keep it as similar as possible to the original commit).

Reply:
But we must be consistent with what we send to the build service. And the version 4.1.10 is the one that include both things. It's not true that there was a 4.1.10 version only with @mvidner's improvement and without the test suite fix.

Reply:
@ancorgs ah yes, looking at it like that, I agree :)